### PR TITLE
Update Cells and gland Segmentation (FRUNet)

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.6865412.
The following version(s) will be added:
  - [6865413 (version 1)](https://www.doi.org/10.5281/zenodo.6865413)

Please review the changes and make sure the new item or version(s) pass the following check list:
- [x] Passed the BioImage.IO CI tests: static (and dynamic) validations
- [ ] The meta information for the RDF item is complete
  - [ ] The tags are complete and describe the model
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D)
  - [ ] Authors are provided
  - [ ] Documentation is complete
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: @Zoufu1122

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: none

### CODE REVIEW

**Comment by oeway** [comment]
@Zoufu1122 There are still some issues in your yaml file needs to be improved before we can merge:
Currently, you have:
```yaml
covers:
- F:/segmentation/kaggle_2018/images/001.png
- F:/segmentation/kaggle_2018/labels/001.png
```

Could you please include the images into the model package and change the path to a relative path? e.g.:
```yaml
covers:
- ./images_001.png
- ./labels_001.png
```
After you prepared the yaml file and cover images, please login to zenodo: https://zenodo.org/record/6865413 and create a new version by uploading the additional files.

Let us know when you finished it, we will check the new version.

**Comment by Zoufu1122** [comment]
@oeway Thanks for your help!
The new version has been uploaded.

**Comment by oeway** [comment]
@Zoufu1122 Thanks for the update!

However, you currently have this in the yaml which won't work on the website:
```
covers:
  - https://zenodo.org/record/6821112/files/image_000.png?download=1
  - https://zenodo.org/record/6821112/files/001.png?download=1
```

The covers should be a relative path, and since you have uploaded image.png and label.png, you can change it to:
```
covers:
 - ./image.png
 - ./label.png
```

Please create another version and update the rdf.yaml file, thanks!

**Comment by Zoufu1122** [comment]
@oeway Thanks a lot! The rdf.yaml file has been updated.

**Comment by oeway** [comment]
@Zoufu1122 Sorry that I forgot to tell you that the weights file should also be changed from:

```
  keras_hdf5:
    source: https://zenodo.org/record/6821112/files/keras_model.h5?download=1
    sha256: 12b23f5a3ab2952109c20770c05539da795b50d79cb1387cacfc14edfbe4684d
    tensorflow_version: 2.6.0
```
to:
```
  keras_hdf5:
    source: ./keras_model.h5
    sha256: 12b23f5a3ab2952109c20770c05539da795b50d79cb1387cacfc14edfbe4684d
    tensorflow_version: 2.6.0
```


@FynnBe Not sure how to handle this, but maybe we should somehow check the URL, the reason is that url like this: https://zenodo.org/record/6821112/files/keras_model.h5?download=1 won't allow us to download from the website due to CORS protection, we need the converted URL with zenodo api.

@esgomezm @FynnBe Do you know why the dynamic testing is skipped? Is there a way to test it?

**Comment by Zoufu1122** [comment]
@oeway Thanks for your help!
Can you help to confirm the content of the yaml file of the latest version？
Because what I see after downloading the yaml file is like this：
  keras_hdf5:
    source: ./keras_model.h5
    sha256: c0f7f23225ed6142f9ac5c7abfd7442afb11181dbe91c936b2d11a620f504655
    tensorflow_version: 2.6.0

[https://zenodo.org/record/6906897#.YuDBK3ZByUk](url)

**Comment by Zoufu1122** [comment]
@oeway Thanks for your help!
Can you help to confirm the content of the yaml file of the latest version？
Because what I see after downloading the yaml file is like this：
keras_hdf5:
source: ./keras_model.h5
sha256: c0f7f23225ed6142f9ac5c7abfd7442afb11181dbe91c936b2d11a620f504655
tensorflow_version: 2.6.0

[https://zenodo.org/record/6906897](url)

**Comment by oeway** [comment]
> @oeway Thanks for your help! Can you help to confirm the content of the yaml file of the latest version？ Because what I see after downloading the yaml file is like this： keras_hdf5: source: ./keras_model.h5 sha256: c0f7f23225ed6142f9ac5c7abfd7442afb11181dbe91c936b2d11a620f504655 tensorflow_version: 2.6.0
> 
> [https://zenodo.org/record/6906897](url)

The new version looks good now, closing this PR to get the newer version.
